store/root.js

```diff
@@ -83,6 +84,33 @@ class Root {
     const { data } = await apiRequest(url, 'GET', {}, {}, true).promise
     Object.assign(this.statistics, data)
   }
+
+  @action
+  async updateDataProvider(patch) {
+    setTimeout(() => {
+      Object.assign(this.dataProvider, patch)
+    }, 300)
+    return
+
+    // TODO: Should call API
+    // eslint-disable-next-line no-unreachable
```

Comment:
As you see, I face actual requests here. This should be changed once API is ready.